doc/user/load_save.md

```diff
@@ -745,74 +1347,72 @@ beforehand.
+This example loads an image, and converts the layout such that channels are
+grouped together in preparation for a convolutional neural network. Then, the
+image is converted back to interleaved channels and saved.
 
 ```c++
 // Download: https://datasets.mlpack.org/images/mlpack-favicon.png
 arma::mat image;
-mlpack::data::ImageInfo info;
-mlpack::data::Load("mlpack-favicon.png", image, info, true);
+mlpack::data::ImageOptions opts;
+opts.Fatal() = true;
+mlpack::data::Load("mlpack-favicon.png", image, info);
 
 std::vector<std::string> colors =
      { "\033[31m", "\033[32m", "\033[34m", "\033[37m" };
 
-// Display input before grouping channels (Load returns channels interleaved).
+// Display input before grouping channels (Load() returns channels interleaved).
 std::cout << "Original Image (channels interleaved):" << std::endl;
-for (size_t i = 0; i < image.n_rows; i += info.Channels())
+for (size_t i = 0; i < image.n_rows; i += opts.Channels())
 {
-  for (size_t j = 0; j < info.Channels(); j++)
+  for (size_t j = 0; j < opts.Channels(); j++)
     std::cout << colors[j] << image.at(i + j, 0) << "\033[0m" << ", ";
 }
 std::cout << std::endl << std::endl;
 
 // Group channels.
-image = mlpack::data::GroupChannels(image, info);
+image = mlpack::data::GroupChannels(image, opts);
 
 // Display submatrix of input after grouping channels
 std::cout << "Grouped channels:" << std::endl;
-for (size_t i = 0; i < info.Channels(); i++)
+for (size_t i = 0; i < opts.Channels(); i++)
 {
-  for (size_t j = 0; j < image.n_rows / info.Channels(); j++)
+  for (size_t j = 0; j < image.n_rows / opts.Channels(); j++)
     std::cout << colors[i] <<
-      image.at(i * image.n_rows / info.Channels() + j, 0) << "\033[0m" << ", ";
+      image.at(i * image.n_rows / opts.Channels() + j, 0) << "\033[0m" << ", ";
 }
 std::cout << std::endl << std::endl;
 
-// Do some computation here, for example a convolutional neural network.
+// Do some computation here; for example, a convolutional neural network.
 
 // Interleave channels to prepare for saving.
-image = mlpack::data::InterleaveChannels(image, info);
+image = mlpack::data::InterleaveChannels(image, opts);
 
-// Display input after interleaving channels
-// Should be identical to original.
-std::cout << "Interleaved channels (indentical to original):" << std::endl;
-for (size_t i = 0; i < image.n_rows; i += info.Channels())
+// Display input after interleaving channels.
+// This should be identical to the original image.
+std::cout << "Interleaved channels (identical to original):" << std::endl;
+for (size_t i = 0; i < image.n_rows; i += opts.Channels())
 {
-  for (size_t j = 0; j < info.Channels(); j++)
+  for (size_t j = 0; j < opts.Channels(); j++)
     std::cout << colors[j] << image.at(i + j, 0) << "\033[0m" << ", ";
 }
 std::cout << std::endl << std::endl;
 
-mlpack::data::Save("mlpack-favicon.png", image, info, true);
+mlpack::data::Save("mlpack-favicon.png", image, opts);
 ```
-
```

Comment:
This deleted line actually causes the next line to not be rendered as a header, so the links to `#letterbox-transform` don't work.  So we'll have to add the empty line back after the code block.

Reply:
I spent like 5 minutes trying to figure this out, and I could not. Thanks for the fix, I will add it.